src/server/components/Sivilstand.tsx

```diff
@@ -7,13 +7,15 @@ interface Props {
 }
 
 const SivilstandGrunnlag: React.FC<Props> = ({ sivilstand }) => {
+  const registergrunnlag = sivilstand.registergrunnlag;
   return (
     <>
       <h3>Registerdata</h3>
-      <div>Sivilstatus: {sivilstandTilTekst[sivilstand.registergrunnlag.type]}</div>
       <div>
-        Gyldig fra og med: {formaterNullableIsoDato(sivilstand.registergrunnlag.gyldigFraOgMed)}
+        Sivilstatus: {sivilstandTilTekst[registergrunnlag.type]}
+        {registergrunnlag.navn && ` med ${registergrunnlag.navn}`}
```

Comment:
Ble nå litt usikker på om det blir riktig å bruke `med` for alle ulike typene.. Mulig man burde ha med/fra. Har sendt spørsmål til Mirja

Reply:
Snakket med Lars, endrer til `-` i stedet for `med
`